crates/client/rpc-core/src/utils.rs

```diff
@@ -44,6 +46,54 @@ pub fn to_rpc_contract_class(contract_class: BlockifierContractClass) -> Result<
     }
 }
 
+/// Returns a [`StateDiff`] from a [`CommitmentStateDiff`]
+pub fn to_rpc_state_diff(commitment_state_diff: CommitmentStateDiff) -> Result<StateDiff> {
```

Comment:
more descriptive name: `blockifier_to_rpc_state_diff_types`

Reply:
It was written like so to comply with the above `to_rpc_contract_class`, should we change both maybe ?

Reply:
Done.